adapta/utils/README.md

```diff
@@ -107,4 +107,44 @@ with memory_limit(memory_limit_percentage=0.2):
     # Do some stuff here
     # if the provided limit is exceeded, MemoryError will be raised and can be caught by client code
     pass
+```
+
+### runtime_metrics:
+
+Decorator for logging method execution time and reporting function execution time to metric provider (DataDog).  
+
+Notes:
+* Function call is logged with loglevel.INFO
```

Comment:
I think this should be configurable

Reply:
Logger will only report messages on DEBUG level now.

Reply:
Why not make it a parameter?

Reply:
Refactored logger reporting level as parameter that you pass in.